### ISSUE TITLE
Huobi Pro: Signature Not Valid with different API URL

### ISSUE THREAD

**Issue by eabrouwer3** [ccxt/ccxt#5617]
- OS: Ubuntu 19.04
- Programming Language version: Python 3.6
- CCXT version: 1.18.983
- Exchange: Huobi/Huobi Pro
- Method: `fetch_balance`

I looked at [this issue](https://github.com/ccxt/ccxt/issues/1485), and part of the way down, there's [this comment](https://github.com/ccxt/ccxt/issues/1485#issuecomment-361530493). I'm wondering if it's possible to have a different API URL and still have it work. I'm a little lost as to why this breaks it.

Basically, I'm trying to adapt the `huobipro` exchange client to work with the normal Huobi.com exchange, because as far as I can see, they work exactly the same (maybe I'm wrong). This is the first issue I've come accross. Any ideas?

I'm passing this in:
```
'urls': {
    'api': {
        'market': 'https://api.huobi.com',
        'public': 'https://api.huobi.com',
        'private': 'https://api.huobi.com',
    }
}
```

**Comment by kroitor** [comment]
> Any ideas?

Try to pass in:

```
'urls': {
    'api': {
        'market': 'https://api.huobi.com',
        'public': 'https://api.huobi.com',
        'private': 'https://api.huobi.com',
    }
},
'hostname': 'api.huobi.com',
```

and let us know if that helps or not.

**Comment by eabrouwer3** [comment]
Perfect! That fixed it. Thanks again for all your help @kroitor!
